CoreSchemaView/release/modules/ext/docs/CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/schema-view-transaction-types.md

```diff
@@ -0,0 +1,11 @@
+
+# Transaction Types
+
+From the Transaction Types tab of the Schema View, you are able to view all the
+key information for all different Transaction types in the schema. 
+
+Information available can include name, description, where it sits in the hierarchy
+of types, and any regular expressions that may be used for detection or
```

Comment:
if we're getting specific with transaction types, they don't have any regular expressions for detection or validation. Could be replaced with whether the type has direction